muspinsim_input/muspinsim_input.xml

```diff
@@ -0,0 +1,423 @@
+                                <option value="transverse">transverse</option>
+                                <option selected="true" value="custom">custom</option>
+                            </param>
+                            <when value="custom">
+                                <param name="polarization" type="text" value="" label="Enter custom vector for polarization"
+                                       help="Accepts 1X3 vector.
+                                       Allows default expressions, constants and functions (see help section)"/>
+                            </when>
+                            <when value="longitudinal" />
+                            <when value="transverse" />
+                        </conditional>
+                    </repeat>
+                </when>
+            </conditional>
+            <param name="average_axes" type="select" display="checkboxes" multiple="true" optional="true" value="orientation" label="Choose keywords that should have an average carried out over them"
+                help="Any keywords not included here or as x-axis are included in separate files">
+                <option selected="true" value="orientation">orientation</option>
+                <option value="polarization">polarization</option>
+            </param>
+            <param name="euler_convention" type="select" display="radio" label="Define euler angle convention (ignored if euler angles not defined)">
+                <option selected="true" value="zyz">zyz</option>
+                <option value="zxz">zxz</option>
+            </param>
+            <repeat name="orientations" title="Orientations to use for crystallites - (define powder averages)">
+                <conditional name="orientation_options">
+                    <param name="orientation_preset" type="select" display="radio" label="Choose orientation type to define" optional="false"
+                           help="Polar angles: Define two polar angles theta and phi, defining only the direction of the z-axis (recommended for powder averages).
+                                Euler angles: Define 3 euler angles defining a new frame, convention used is ZYZ by default.
+                                Euler angles with weight: define 3 euler angles and a weight (will be normalized automatically).
+                                Euler angles helper function: eulrange(n).
+                                Zaremba-Conroy-Wolfsberg helper function: zcw(n)">
+                        <option value="2_polar">Polar angles</option>
+                        <option value="3_euler">Euler angles</option>
+                        <option value="4_euler">Euler angles with 1 weight</option>
+                        <option value="eulrange">Euler angles helper function (eulrange(n))</option>
+                        <option value="zcw">Zaremba-Conroy-Wolfsberg helper function (zcw(n))</option>
```

Comment:
Leandro tells me that zcw is the most commonly used setting as it's a convenient helper function, so this should be at the top and set as default.